src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/latest/test_storage_container_immutability_policy.py

```diff
@@ -0,0 +1,56 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import (ScenarioTest, JMESPathCheck, ResourceGroupPreparer, StorageAccountPreparer)
+from msrestazure.azure_exceptions import CloudError
+
+
+class StorageImmutabilityPolicy(ScenarioTest):
+    @ResourceGroupPreparer()
+    def test_immutability_policy(self, resource_group):
+        storage_account = self.create_random_name('clistorage', 20)
+        self.cmd('storage account create -g {} -n {} --kind StorageV2'.format(
+            resource_group, storage_account))
+        container_name = 'container1'
+        self.cmd('storage container create --account-name {} -n {}'.format(storage_account, container_name))
```

Comment:
What's the difference between prepending `az` in the test command and not doing so? I am assuming if one doesn't add `az` to the test command, it is inferred?

Reply:
Yes, since every command will start with `az`. I generally try not to add the 'az ' as it is redundant.